Java/src/main/java/Planes/experimentalPlane.java

```diff
@@ -5,12 +5,12 @@
 
 public class experimentalPlane extends Plane{
```

Comment:
Class names are  UpperCamelCase in Java
https://google.github.io/styleguide/javaguide.html#s5.2.2-class-names